Pipfile

```diff
@@ -17,6 +17,7 @@ quandl = "==3.4.8"
 statsmodels = "==0.10.1"
 bs4 = "==0.0.1"
 tweepy = "==3.5.0"
+pandas = "*"
```

Comment:
Would you be able to pin this version as above please? You can get the current version with `pip freeze` and then you can regenerate the lock file with `pipenv lock`